src/Formatters/VarDumpFormatter.php

```diff
@@ -37,4 +38,14 @@ public function write(OutputInterface $output, $data, FormatterOptions $options)
             fclose($stream);
         }
     }
+
+    public function isValidDataType(\ReflectionClass $dataType)
+    {
+        return true;
```

Comment:
What does OutputFormatters do without the ValidationInterface? It does seem that `VarDumpFormatter` and `VarExportFormatter` should work with any data type, so we could put this change into OutputFormatters.

Reply:
Without ValidationInterface a formattter is eligible for array and null only.